rero_ils/modules/budgets/api.py

```diff
@@ -67,16 +67,18 @@ class Budget(IlsRecord):
         }
     }
 
-    def get_number_of_acq_accounts(self):
+    def get_number_of_acq_accounts(self, get_pids=False):
```

Comment:
the method name suggests that we get the number of linked acq_account. Either change the signature, either don't play with `get_pids`